101_Pythonvariablen.ipynb

```diff
@@ -34,7 +34,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Programmieren besteht aus einer Aneinanderreihung von Befehlen, die der Python Interpreter ausführt. Ein einfacher Befehl ist die Zuweisung eines Wertes zu einer **Variablen**:"
+    "## Über Variablen\n",
+    "\n",
+    "Pythoncode besteht aus einer Aneinanderreihung von *Befehlen*, die der sogenannte Python-Interpreter _nacheinander_ ausführt. Mit anderen Worten geht der Interpreter Zeile für Zeile durch ein Python-Script oder, wie in unserem Fall, durch eine Jupyter-Codezelle und übersetzt das, was in der jeweiligen Zeile steht in elementare \"Operationen\", die dann auf der CPU ausgeführt werden können.\n",
+    "\n",
+    "Ein solcher Befehl ist die Zuweisung eines Wertes zu einer **Variablen**. Dabei übersetzt der Interpreter den Wert der Variablen in Nullen und Einsen und schreibt diese an einen Ort im Arbeitsspeicher. Und gleichzeitig bekommt der Ort einen Namen, den wir uns Menschen gut merken können, damit wir später wieder auf diese Variable zugreifen können.\n",
+    "\n",
+    "Im unteren Beispiel weisen wir der Variablen mit dem Namen `x` den Wert `1` zu."
```

Comment:
ich würde hier nicht 1 nehmen, sondern whatever, 42. Aber um potenzielle Verwirrung mit der Formulierung "wird in 0 und 1 übersetzt" von oben zu vermeiden. Vllt sogar noch nen Satz, was die binäre Übersetzung davon wäre

Reply:
Gute Idee! Mach ich!